src/routes/InstancesPage/DeleteInstanceModal.js

```diff
@@ -27,6 +27,17 @@ function DeleteInstanceModal({ isOpen, instance, onRequestDelete, onClose }) {
     }
   }
 
+  function deleteInstanceOnSubmit(e) {
+    e.preventDefault();
+    if (inputMatchesInstanceName()) {
+      onRequestDeleteHandler();
+    }
+  }
+
+  function inputMatchesInstanceName() {
+    instance.name === inputValue;
```

Comment:
I think you forgot to return the value
```suggestion
    return instance.name === inputValue;
```

Reply:
Oops, tried to one line it and eslint pulled it back down. Good catch

Reply:
I also could've just committed your suggestion 🙃

Reply:
Oh, wait did ESLint not catch that error?

Reply:
I assume it's technically not considered an error. A function without a `return` just returns undefined